debian/nautilus.postinst

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+XDG_MIMEAPPS_FILE=/etc/xdg/mimeapps.list
+XDG_MIMEAPPS_HEADER="[Default Applications]"
+
+if [ "$1" = configure ]; then
+    # First non-empty line should be the required header, otherwise the file
+    # will be considered corrupt and will be removed to be filled later.
```

Comment:
Editing an INI file in shell. Not fun...

Reply:
Yeah, I know... I checked into doing this in python, but I kind of came to the conclusion reading comments out there that for maintainers scripts shell is kind of the standard, so went for it.

I'm open to change it to python if that's not too much of a blasphemy, though :)

Reply:
I think it's good enough. I just don't envy you :)